src/chef-server-ctl/bin/chef-server-ctl

```diff
@@ -28,13 +29,13 @@ module Omnibus
       d
     end
 
-    CREDENTIAL_ENV = "CHEF_SECRETS_DATA".freeze
+    CREDENTIAL_ENV = "#{ChefUtils::Dist::Infra::SHORT.upcase}_SECRETS_DATA".freeze
 
     def credentials
       if ENV.has_key?(CREDENTIAL_ENV)
         @credentials = Veil::CredentialCollection::ChefSecretsEnv.new(var_name: CREDENTIAL_ENV)
       else
-        secrets_file = ENV["SECRETS_FILE"] || "/etc/opscode/private-chef-secrets.json"
+        secrets_file = ENV["SECRETS_FILE"] || "/etc/opscode/private-#{ChefUtils::Dist::Infra::SHORT}-secrets.json"
```

Comment:
Yes, but maybe this was me going too far. I am not sure if configuration files like this need to have the trademark/workmarks removed from their path names?

Reply:
IMO this should stay and also fix the `opscode` reference to use the constant for `LEGACY_CONF_DIR`

Reply:
These are directory names, so if you change those, things will break unless you also change the directories themselves.

Reply:
From what I understand https://github.com/chef/chef/blob/main/chef-utils/lib/chef-utils/dist.rb#L43 will make it so that there are no changes for Chef builds.